### ISSUE TITLE
Final Review

### ISSUE THREAD

**Issue by OliviaAbbott** [tanyabonilla/portfolio#6]
# Summary

The project was looking at airbnb data, initially the goal was to look at the airbnb data in comparison to hotel data in New York City.  After not being able to find hotel data, the project then focused on looking at what different things (like number of bedrooms) were affecting the airbnb prices.  The project found that there was a relationship between price and the number of bedrooms.  The next steps were finding the hotel data that could be used to try to see what relationships there are between airbnbs and hotels.

# Data Preparation

The project uses two different sources of data.  The first data source in deliverable 1 holds the airbnb data.  This data has information on the host, the reviews, and the listing.  I thought that this data was cleaned really well and follows the tidy data principles.  The data was well explained and splitting the initial dataset into the three tibbles was a good choice.  The second dataset was a New York airbnb listings dataset that contained a lot of data.  This dataset was narrowed down to hold information on just the airbnb property attributes, location data, and price information.  This dataset was a little harder for me to understand.  At first I did not realize that it was airbnb data, I had to use the link in deliverable 2 to look at where the data came from before I realized that it was only holding airbnb data.  I think the data was also cleaned well and followed tidy data principles. It was also split into three tibbles, similarly to the dataset from deliverable 1. My only suggestion for improvement would be to add a little more of an explanation in deliverable 2 on where this dataset is coming from, what the dataset contains, and how it relates to the first deliverable.

# Modeling

In this project, two different models are created.  Both models use the data from the second dataset.  The first uses the bedroom and bathroom data to predict the price of the airbnb.  The second model uses neighborhood data, the type of bed data, and the availability of the listing data to predict the price of the airbnb.  I think that the project explains the purpose behind each model well.  The first model is explained in more detail and more analysis is provided on the model's results.  The first model explains the model's summary and gives good explanations for the p-value and what that means for the model.  The second model is a little harder to follow.  The model is created twice, once using all of the data and the second time only using the training data.  I am a little confused as to why it is done twice since that is not explained. Also on the second model the results are displayed but they are not explained as well as the first model.

# Validation

The validation aspect of the project I think is done well on the model in deliverable 2.  The model is created and the test set is used to create predictions.  Using those predictions the R2 and MAE values are calculated and explained.  The model in deliverable 3 also uses the test predictions to calculate those values, but they are not explained so it is harder to know what this model means or if it was very accurate.

# R Proficiency

The R code in this project I found very easy to read.  I was able to follow along with the code well.  Everything is explained well and it was very clear to me the purpose behind all the different sections of code.  There is only one part of the project that would be harder to maintain and reuse.  In deliverable 2, in order to get the data you would need to download it yourself since it wasn't able to be added to GitHub.  Although this part of the project would make it harder to reuse, it is well explained and understandable.

# Communication

I think that the overall communication in this portfolio is very well done. Deliverable 1 and 2 were very easy for me to follow and everything is well explained.  I found that it was easy to see what was trying to be done and the analysis in these two sections are thorough.  The communication and explanations in deliverable 3 were not as well done.  While there were explanations for what was trying to be done in the section, the results were not as fully explained so I was not able to as easily understand the model in this section.

# Critical Thinking

The critical thinking is well done in this project.  At the end of deliverable 3 there is a section that talks about the future steps as well as some of the social impacts and consequences.  The project also considers the consequences once some hotel data is found and used in the project.  The only thing that I thought was missing was some action that could be taken as result of this project.  Perhaps airbnb could uses the data in this project to suggest prices for people who are renting on their site.  This way people who rent out would have a better idea of what their property rental should cost given factors such as the neighborhood, number of rooms, etc.

**Comment by tanyabonilla** [comment]
# Data Preparation and Modeling (15 out of 20%)

*Rationale for score*
I prepared the data and models using the "train" data and the normal data set to find if there were any differences between each model. I prepared the data to fit what I was trying to predict and the questions I wanted to answer; for the most part.

# Validation and Operationalization (15 out of 20%)

*Rationale for score*

I included cross-validated two of my models and made an attempt to explain what was happening. I think that my what could be done with the data could be better. 

# R Proficiency (15 out of 20%)

*Rationale for score*
I am proficient in R, but there is definitely room for improvement as I feel like I used basic functions and thoughts. 

# Communication (15 out of 20%)

*Rationale for score*
I try to explain what is going on throughout, but all short in explaining my first model.

# Critical Thinking (20 out of 20%)

*Rationale for score*
I tried and feel like succeeded in thinking critically of the consequences that could come out of the the data that is found.
